src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -171,16 +195,15 @@ private static CliCommand ConstructCommand()
 #if RELEASE
```

Comment:
@MarcoRossignoli , do you think we should remove this condition (if release) once we are done with the dotnet test experience?

Reply:
We can remove the env var.